### PR TITLE
Move skipCoverage to the config

### PR DESCRIPTION
**Pull request by theofidry**

### CODE REVIEW

File: src/Configuration/ConfigurationFactory.php [hunk 1]
```diff
@@ -201,18 +207,18 @@ private static function retrieveExistingCoverageBasePath(
         ?string $existingCoveragePath,
         string $configDir,
         string $tmpDir
-    ): string {
```
Comment (BackEndTea) [inline src/Configuration/ConfigurationFactory.php]: Does phpstan understand annotations like: ` @return array{0:bool,1:string}`?

Comment (theofidry) [reply src/Configuration/ConfigurationFactory.php]: At least it doesn't seem to complain about it 👍

File: src/Configuration/ConfigurationFactory.php [hunk 2]
```diff
@@ -113,6 +113,12 @@ public function create(
 
         $testFramework = $testFramework ?? $schema->getTestFramework() ?? TestFrameworkTypes::PHPUNIT;
 
+        [$skipCoverage, $existingCoverageBasePath] = self::retrieveExistingCoverageBasePath(
```
Comment (maks-rafalko) [inline src/Configuration/ConfigurationFactory.php]: IMO, now the method name conflicts with return values. It doesn't return path, but some flag as well.

What about setting it outside the method as:

```php
$skipCoverage = $existingCoverageBasePath !== null;
```

Comment (theofidry) [reply src/Configuration/ConfigurationFactory.php]: 👍

**Pull request merged**